core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -89,7 +89,7 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertFalse(config.useNewConsumer)
+    assertTrue(config.useOldConsumer)
```

Comment:
Doesn't the old assertion work any more? We should probably keep it to ensure that we are compatible.